be/src/common/config.h

```diff
@@ -1396,4 +1396,10 @@ CONF_mString(try_release_resource_before_core_dump, "data_cache");
 // Experimental feature, this configuration will be removed after testing is complete.
 CONF_mBool(lake_enable_alter_struct, "true");
 
+CONF_mInt32(thrift_max_message_size, "1073741824"); // 1 GB
```

Comment:
better add some comments explaining a little bit about the parameters and possibly under what condition, may consider tune these parameters.